jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -145,6 +149,33 @@ public static JarType determineJarType(Path jarPath) throws IOException {
     return layers;
   }
 
+  /**
+   * Computes the entrypoint for a standard jar in exploded mode.
+   *
+   * @param jarPath path to jar file
+   * @param tempDirPath path to temporary jib local directory
+   * @param layers list of {@link FileEntriesLayer}
+   * @return list of {@link String} representing entrypoint
+   * @throws IOException if I/O error occurs when opening the jar file or if temporary directory
+   *     provided doesn't exist
+   */
+  protected static ImmutableList<String> computeEntrypointForExplodedStandard(
+      Path jarPath, Path tempDirPath, List<FileEntriesLayer> layers) throws IOException {
+    Path localExplodedJarRoot = tempDirPath;
+    ZipUtil.unzip(jarPath, localExplodedJarRoot);
+
+    String mainClass = null;
+    try (JarFile jarFile = new JarFile(jarPath.toFile())) {
+      mainClass = jarFile.getManifest().getMainAttributes().getValue(Attributes.Name.MAIN_CLASS);
+    }
```

Comment:
Nit: I think this code block is super fast, so it's good to put everything in to the try-resource block.
```suggestion
    try (JarFile jarFile = new JarFile(jarPath.toFile())) {
      String mainClass = jarFile.getManifest().getMainAttributes().getValue(Attributes.Name.MAIN_CLASS);
      if (mainClass == null)
      ...
      return ...
    }
```

Reply:
Done.